src/Faker/Provider/fr_FR/Company.php

```diff
@@ -166,4 +165,16 @@ protected static function isCatchPhraseValid($catchPhrase)
 
         return true;
     }
+
+    /**
+     * @link http://www.emploi.com/emploi/index-metiers.html
```

Comment:
😓 I understand that and didn't check. It's actually just a list of job titles in French... How can it be "reserved"?

Anyway, removing the link will make the difference between this list and a list made by hand... But before that, I have asked the site if they can give us the permission to reuse the list. Would it be OK ?